nautobot_golden_config/templates/nautobot_golden_config/goldenconfigsetting.html

```diff
@@ -1,248 +1,131 @@
-                            <tr>
-                                <td>
-                                    Jinja Repository
-                                </td>
-                                <td>
-                                    {% if object.jinja_repository %}
-                                        <a href="{{ object.jinja_repository.get_absolute_url }}">{{ object.jinja_repository }}</a>
-                                    {% else %}
-                                        <span class="text-muted">None</span>
-                                    {% endif %}
-                                </td>
-                            </tr>
-                            <tr>
-                                <td>
-                                    Template Path in Jinja Template Form
-                                </td>
-                                <td>
-                                    {{ object.jinja_path_template|placeholder }}
-                                </td>
-                            </tr>
-                            <tr>
-                                <td>GraphQL Query</td>
-                                <td>
-                                    {% if object.sot_agg_query %}
-                                        <a href="{{ object.sot_agg_query.get_absolute_url }}">{{ object.sot_agg_query }}</a>
-                                    {% else %}
-                                        <span class="text-muted">None</span>
-                                    {% endif %}
-                                </td>
-                            </tr>
-
-                        </table>
-                    </div>
-                </div>
-            </div>
-
-            <div class="row">
-                <div class="col-md-12">
-                    {% block content_full_width_page %}{% endblock content_full_width_page %}
-                    {% plugin_full_width_page object %}
-                </div>
-            </div>
-        </div>
-        <div id="advanced" role="tabpanel" class="tab-pane {% if request.GET.tab == 'advanced' %}active{% else %}fade{% endif %}">
-            <div class="row">
-                <div class="col-md-6">
-                    {% include 'inc/object_details_advanced_panel.html' %}
-                </div>
-                <div class="col-md-6">
-                    {% block advanced_content_right_page %}{% endblock advanced_content_right_page %}
-                </div>
-            </div>
-            <div class="row">
-                <div class="col-md-12">
-                    {% block advanced_content_full_width_page %}{% endblock advanced_content_full_width_page %}
-                </div>
-            </div>
-        </div>
-    </div>
-{% endblock content %}
-
-{% block javascript %}
-<script>
-    url = window.location.href.split("?")[0];
-    window.history.pushState({}, document.title, url);
-    var clipboard = new ClipboardJS('.btn');
```

Comment:
Did this not remove the copy text file? I was looking to replace this anyway, but don't see how that is working here. 
https://github.com/nautobot/nautobot-plugin-golden-config/issues/287

Reply:
I will check this.

Reply:
I was expecting to see `hover_copy_button`, how did you handle the copy here?

Reply:
Copy of what? This is just the setting page not seeing where a copy icon shows up.

Photo from kitt
![image](https://user-images.githubusercontent.com/26559188/182677932-88e51f65-e694-4ff3-a7a9-1f178088b13b.png)

Reply:
I believe the removed JS was an artifact from copy paste and not actually used on this view.